packages/theming/theme-tokens/src/highContrast/tokens-alias.ts

```diff
@@ -0,0 +1,18 @@
+import { PlatformColor } from 'react-native';
+import aliasTokens from '@fluentui-react-native/design-tokens-win32/hc/tokens-aliases.json';
+
+export const hcAliasTokens = processAliasTokens(aliasTokens);
+
+function processAliasTokens(aliasTokens: any) {
+  for (const key in aliasTokens) {
+    for (const innerKey in aliasTokens[key]) {
+      const entry = aliasTokens[key][innerKey];
+      if (typeof entry === 'string' && entry.includes('PlatformColor')) {
+        const color = 'SystemColor' + entry.substring(14, entry.length - 1) + 'Color';
```

Comment:
This looks like possibly Android specific logic?

Reply:
No, this is existing code from "default-themes" where getTokens.ts uses this,  getTokens.ts is moved in this package, this folder is also refactored here. 

So I need to have platform implementations for android for the API's used in "[createAliasTokes.ts](https://github.com/microsoft/fluentui-react-native/pull/2159/files#diff-0066b5262771ef012f09b7c921851dc60560a7f5a23234fd2d7b0b73817f1682)" for android -
 1)  aliasTokens = `getAliasTokens`(mode);

This api returns the tokens based on light and dark mode for android based on appearance ( no high contrast mode). This is a subset of theme tokens for other endpoints.
The alternative was to keep a file 'getTokens.ts' in every folder, but I moved getTokens to be exported from theme-tokens packages as:
1.1 ) The flow in fluent tester triggers default theme flow - IDK why yet, but not having a android implementation for this mixes flow to pick "./getTokens" api for default themes and then calls 
`mapPipelineToTheme`(aliasTokens) implemented for android - crashes as token set is not 1-1. So there needs a common package export for this api which is theme-tokens.

1.2) I felt theme-tokens is fitting right, this exports globals - and having "./getTokens" in same folder keeps all the code referring to design-tokens of fluentui-desing-tokens directly in same package.

 2) `mapPipelineToTheme`(aliasTokens); 
This needed a fork for android, as there are tokens not present in android. For eg, 
    transparentStroke: **pipelineOutput.transparentStroke**.**strokeColorRest**,
    transparentStrokeInteractive: **pipelineOutput.transparentStroke**.**strokeColorHover**,
    transparentStrokeDisabled: **pipelineOutput.transparentStroke**.**strokeColorDisabled**,

The **bold** ones are not by design in android, so passing through default `packages/theming/theming-utils/src/mapPipelineToTheme.ts` crashes for multiple such tokens. The android corresponding file added PR only has the tokens which are meaningful for android.  I considered using optional chaining in the file, but this looked more readable.

The need is to allow decoupling of tokens code between existing desktop and android, if there are alternate options on how to achieve this, keen for suggestions.